akka-remote/src/main/scala/akka/remote/serialization/MiscMessageSerializer.scala

```diff
@@ -66,15 +76,22 @@ class MiscMessageSerializer(val system: ExtendedActorSystem) extends SerializerW
 
   private val IdentifyManifest = "A"
   private val ActorIdentifyManifest = "B"
+  private val SomeManifest = "C"
+  private val NoneManifest = "D"
```

Comment:
use one manifest, `OptionManifest` the fact that it's None or Some is derived from if the `payload` field is defined or not